qiskit/providers/backend_compat.py

```diff
@@ -266,19 +266,19 @@ def dtm(self) -> float:
 
     @property
     def meas_map(self) -> List[List[int]]:
-        return self._config.dt
+        return self._config.meas_map
 
     def drive_channel(self, qubit: int):
-        self._config.drive(qubit)
+        return self._config.drive(qubit)
```

Comment:
Lol, this is my bad. I switch back and forth between rust and python too much (Rust has an implicit return on the last line of a function without a semicolon)

Reply:
Certainly, the reason is very persuasive!